test/fhirtypes/Range.test.ts

```diff
@@ -0,0 +1,112 @@
+import cloneDeep from 'lodash/cloneDeep';
+import { Range, validateRange } from '../../src/fhirtypes/Range';
+import { Quantity } from '../../src/fhirtypes/Quantity';
+import { InvalidRangeValueError } from '../../src/errors/InvalidRangeValueError';
+import { UnitMismatchError } from '../../src/errors/UnitMismatchError';
+import { CodeAndSystemMismatchError } from '../../src/errors/CodeAndSystemMismatchError';
```

Comment:
We can clean up these imports a bit once we add these types to their respective `index.ts` files.